GameMakerHub/Catalyst
# Upgrade a test project, and implement local directory repositories

### Fix #12 
Link to ticket: https://dukesoft.atlassian.net/browse/cat-12

### What has been done
- Implement a local directory repository

### How to test
- Initialize a catalyst project somewhere
- Open up another catalyst project, and require that first project
- Add the parent folder as a repository in your `catalyst.json`:
```
{
    "name": "dukesoft/gml-project",
    "description": "A testing project with some contents",
    "license": "MIT",
    "homepage": "http://www.dukesoft.nl",
    "yyp": "GMLProject.yyp",
    "repositories": {
        "directory": "../../projects/"
    },
    "require": {
        "dukesoft/other-project": "*"
    }
}
```
- Run install and see that both online repo packages are installed as well as local directory projects
